rush.json

```diff
@@ -716,6 +716,9 @@
     {
       "packageName": "@azure/storage-blob",
       "projectFolder": "sdk/storage/storage-blob",
+	  "cyclicDependencyProjects": [
```

Comment:
nit: indentation is incorrect.

not blocking: would it be possible to use arm package to achieve the same goal so we don't have cyclic dependency?

Reply:
The dependency is only for test environment,  product packages don't have cyclic dependency.  We just need to create several files in blob testing, and would need to create blobs in file testing.. arm packages cannot create file/blobs..